### PR TITLE
[Doc Feature] TeamsJS support matrix across M365 hosts

### PR DESCRIPTION
**Pull request by erikadoyle**
Adding an article which details Teams/M365 app/Outlook support for different TeamsJS capabilities. The support tables are backed by the [TeamsJS Capability Checker](https://github.com/vikramtha/microsoft-teams-library-js/tree/vikramtha/teamsjs-cc-app/apps/teamsjs-cc-app) sample app ([PR in progress](https://github.com/OfficeDev/microsoft-teams-library-js/pull/1660) to add to the TeamsJS source). We are planning to update the article on a regular cadence (monthly to start with, as host support doesn't typically come online faster than that).

This PR is a continuation of PR#8386, and it addresses all of the open issues from previous engineering reviews.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3509f58503cc9382ea27d0629b3ab019e52e030e/msteams-platform%2FTOC.yml) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b56b6f93-c7b4-4fbc-a23f-9296ac20bf90) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3509f58503cc9382ea27d0629b3ab019e52e030e/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bb6df625-127f-4698-b9d3-fd07c09f39fd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3509f58503cc9382ea27d0629b3ab019e52e030e/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ad15fd25-d7ae-431c-bafc-d87ab5ae34f6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/5b32d1dc904953327b930e719af951b3b7422746/msteams-platform%2FTOC.yml) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/441c0efc-6573-49b5-acfd-07762ddc0377) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5b32d1dc904953327b930e719af951b3b7422746/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e03db7d2-abbb-475f-ade2-52e3b9679bce) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5b32d1dc904953327b930e719af951b3b7422746/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bae3e914-1e0b-4e75-bc0a-1efb916a8168) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e2bf9a50fc19db592198155daa0ba0f31587ea44/msteams-platform%2FTOC.yml) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/53de4580-273b-4c02-9b9f-1a31af91834c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e2bf9a50fc19db592198155daa0ba0f31587ea44/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 88 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/735a7313-ecaf-4026-8c21-6ed5cb59be52) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e2bf9a50fc19db592198155daa0ba0f31587ea44/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d6640044-6995-4292-be9c-552418810fa9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/6b0fb5599017ef2611316dab231cb6fbf015e0f5/msteams-platform%2FTOC.yml) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5cc5e5a4-536b-42c8-822b-3f19ffa104a7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6b0fb5599017ef2611316dab231cb6fbf015e0f5/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 88 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0b3580a8-b22c-4a2d-bd1e-c98ee7e3c281) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6b0fb5599017ef2611316dab231cb6fbf015e0f5/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9f72b8bc-7f0d-4ce1-be18-3c88925cdbbe) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1dcdc3ab11245ac2269cd6068b431a753ef1032b/msteams-platform%2FTOC.yml) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2ae409a2-ca7c-4549-bd38-40a61f8499ea) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1dcdc3ab11245ac2269cd6068b431a753ef1032b/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 88 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b7085890-4eff-4fb9-9ea9-81930d4dda65) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1dcdc3ab11245ac2269cd6068b431a753ef1032b/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9889c0a9-77af-4d65-8dac-98548ef68a85) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/85455b1a9d6fd037813a5a482e48c06dc9f7a4ac/msteams-platform%2FTOC.yml) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0196abd1-0543-401c-86e0-db9e380263c9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/85455b1a9d6fd037813a5a482e48c06dc9f7a4ac/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 88 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b39c2209-551a-49b9-ba96-553bf35e6715) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/85455b1a9d6fd037813a5a482e48c06dc9f7a4ac/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/90d77d98-b139-48e9-8fc0-a026c1d1414d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by erikadoyle** [comment]
@surbhigupta12 this is ready from our side; could someone on your team please help review/merge? Thanks!

**Review by TrevorJoelHarris — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4a0220eb94840ee9025e64f19914c37984943a41/msteams-platform%2FTOC.yml) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dd60c382-280b-4d6a-b986-2c08526649d0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a0220eb94840ee9025e64f19914c37984943a41/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 88 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9cb58f75-4fbb-43c4-8b1d-b094926485f0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a0220eb94840ee9025e64f19914c37984943a41/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9b2f0494-7045-4ee3-965d-af2802db240d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/teamsjs-support-m365.md [hunk 1]
```diff
@@ -0,0 +1,2365 @@
...
+            <td>&#x2713;</td>
+            <td></td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
+
+## `pages`
+
+[Reference](/javascript/api/@microsoft/teams-js/pages) | [Sample](https://github.com/vikramtha/microsoft-teams-library-js/blob/vikramtha/teamsjs-cc-app/apps/teamsjs-cc-app/tabs/src/components/capabilities/Pages.tsx)
```
[the remark this answers is not in the document]
Comment (erikadoyle) [reply msteams-platform/m365-apps/teamsjs-support-m365.md:1556]: Just temporarily until the PR to TeamsJS source is completed. @vikramtha is keeping it updated as he addresses review feedback.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/30fc13b4463805572316913da5c9169eb5f6cc26/msteams-platform%2FTOC.yml) | 99 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8b1869e7-aa10-4f1d-af84-cc40b1555281) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/30fc13b4463805572316913da5c9169eb5f6cc26/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 88 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2359afb0-4cbe-4fd1-8501-81748c4af623) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/30fc13b4463805572316913da5c9169eb5f6cc26/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/641a127f-2842-44e6-913e-31814b49b7d4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]

**Pull request merged**